docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -1,87 +1,77 @@
 openapi: 3.0.0
 info:
-  title: Company
-  version: '1.0'
-  description: Company refers to a collection of customers. BigCommerce B2B Edition special provided company member management and convenience of company purchasing.
+  title: Companies and Users
+  description: |-
+    ## Introduction
```

Comment:
Just a general comment on information organization here:  I'm not sure the REST Storefront API is the best place to have intro information for various entities.  As far as I know, the GraphQL API is recommended over the REST API, so it probably doesn't make sense for this to be the "main" entry point for conceptual information like this.

Feels like a separate docs section would be better, which could then link to the various APIs dealing with these entities.  Barring that, I feel like the REST Management API might be a better "entry point."